addon/components/bourbon-toggle.js

```diff
@@ -1,58 +1,40 @@
 import Component from '@ember/component';
+import { computed } from '@ember/object';
 import layout from '../templates/components/bourbon-toggle';
 
 export default Component.extend({
   layout,
   tagName: 'label',
   classNames: ['BourbonToggle'],
+  classNameBindings: ['disabled:BourbonToggle--disabled'],
   attributeBindings: ['label:aria-label'],
 
-  init() {
-    this._super(...arguments);
-    this.send('setToggleState')
-  },
   ariaRole: 'button',
-
   value: null,
-  toggleState: null,
   disabled: false,
   action: null,
-  readOnly: false,
-  label: 'toggle button',
+
+  label: computed('title', 'value', 'disabled', function() {
```

Comment:
🍹 Do we need to preserve the `toggle button` in the aria label? Just wondering what we want as the end result to be read out by screen-readers.